package.json

```diff
@@ -117,7 +117,7 @@
     "idb-keyval": "2.2.0",
     "intl-messageformat-parser": "^1.8.1",
     "isomorphic-fetch": "^2.2.1",
-    "jest": "^24.3.0",
+    "jest": "^24.9.0",
```

Comment:
anything in particular this buys us?

Reply:
was looking into this yesterday...


they have way better messages for failed reports now. and inline snapshots are indented with the code, so look way better (won't update our existing snapshots until something changes in the snapshot, tho)

Reply:
The package.json change wasn't intentional. I installed jest locally and accidentally updated package.json :neutral_face: